ts/nni_manager/common/utils.ts

```diff
@@ -449,8 +450,49 @@ function withLockSync(func: Function, filePath: string, lockOpts: {[key: string]
     return result;
 }
 
+async function isPortOpen(host: string, port: number): Promise<boolean> {
+    return new Promise<boolean>((resolve, reject) => {
+        try{
+            const stream = net.createConnection(port, host);
+            const id = setTimeout(function () {
```

Comment:
Please use `() => { }` to make code style unified.

Reply:
fix it